ci-operator/config/openshift/assisted-test-infra/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- tsorya
```

Comment:
eran

Reply:
shouldnt it be part of the assisted-test-infra repo?

Reply:
> eran

Isn't a member of OpenShift organization.

> shouldnt it be part of the assisted-test-infra repo?

Maybe in the future. Right now Prow needs an OWNERS file for each directory.